src/commands/integration/deploy.ts

```diff
@@ -125,7 +136,6 @@ async function toCreateIntegrationRequest(
       spec.provider.url,
       spec.provider.method,
       spec.provider.headers,
-      spec.provider.timeout,
```

Comment:
It would be better to show deprecation error when user gives timeout.
User might expect that `timeout` works.